web/index.html

```diff
@@ -43,16 +43,22 @@ <h2 class="f4">Where do I find my multiaddr?</h2>
             <li class="pb2">
                 <strong>Using IPFS Desktop or IPFS WebUI</strong>
                 <ul>
-                    <li>Open the IPFS WebUI "Status" page via the IPFS Desktop menu or by visiting "http://127.0.0.1:5001/webui" (when using the default config settings)
-                    <li>Click the "Advanced" dropdown to see more detailed node settings</li>
+                    <li>Open the IPFS WebUI "Status" page via the IPFS Desktop menu or by visiting "http://127.0.0.1:5001/webui" (when using the default config settings)</li>
+                    <li>If you want to test your peerID rather than a particular address enter <code>/p2p/{YourPeerID}</code></li>

```

Comment:
While we could make it so you can just dump in a peerID instead of `/p2p/peerID` (either on the client or server side) expressing this as a multiaddr might be a little clearer to people.

Does the pain of typing in `/p2p/` before copy pasting make this not worth while?

Reply:
I think it's worth it to help get people used to multiaddresses and not expect this behaviour ... though it does seem pretty convenient to just give it a peerID. I do think it should remain a multiaddress though, get people used to the idea of a distinct separation from multiaddr and peerID.